src/lib/datepicker/calendar.ts

```diff
@@ -111,6 +112,7 @@ export class MdCalendar<D> implements AfterContentInit {
   constructor(private _elementRef: ElementRef,
               private _intl: MdDatepickerIntl,
               private _ngZone: NgZone,
+              @Optional() @Inject(MATERIAL_COMPATIBILITY_MODE) public _isCompatibilityMode: boolean,
```

Comment:
can't be private since it's used in the template, underscore indicates that its for internal use only